pkg/installmanager/installmanager.go

```diff
@@ -513,6 +516,18 @@ func (m *InstallManager) generateAssets(provision *hivev1.ClusterProvision) erro
 		}
 	}
 
+	if src := m.ManifestsMountPath; isDirNonEmpty(src) {
+		m.log.Info("copying user-provided manifests")
+		dest := filepath.Join(m.WorkDir, "manifests")
+		out, err := exec.Command("bash", "-c", fmt.Sprintf("cp %s %s", filepath.Join(src, "*"), dest)).CombinedOutput()
+		fmt.Printf("%s\n", out)
```

Comment:
intentional print? Looks like it might be. Should it go through the logger though?

Reply:
Yes. This was intentional to get the error output if the copy failed. I did not run it through the logger because it may be multiline, which does not output well.